script/glam/export_csv

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# To be run in glam via:
+#    docker-compose run server ./manage.py import_glean_aggs
+
+set -e
+
+src_project=${SRC_PROJECT:-glam-fenix-dev}
+# TODO: glam-dev-bespoke-nonprod-dataops-mozgcp-net
+dst_project=${DST_PROJECT:-$src_project}
+dataset=${DATASET:-glam_etl_dev}
+product=${PRODUCT:-fenix}   # TODO: set default to org_mozilla_fenix
+
+bucket="gs://${dst_project}"
+gsutil ls "$bucket" > /dev/null
+
+bq extract --destination_format CSV \
+    "${src_project}:${dataset}.${product}_client_probe_counts_extract_v1" \
+    "$bucket/glam-extract-${product}-*.csv"
+
+bq extract --destination_format CSV \
+    "${src_project}:${dataset}.${product}_user_counts_extract_v1" \
+    "$bucket/glam-extract-${product}-counts.csv"
```

Comment:
Does this default to printing headers in the CSVs?

Reply:
Yes this does, I've added the appropriate flags to stop printing the header.